src/oas.md

```diff
@@ -322,7 +324,7 @@ Relative references in CommonMark hyperlinks are resolved in their rendered cont
 API endpoints are by definition accessed as locations, and are described by this specification as **_URLs_**.
 
 Unless specified otherwise, all fields that are URLs MAY be relative references as defined by [RFC3986](https://tools.ietf.org/html/rfc3986#section-4.2).
-Unless specified otherwise, relative references are resolved using the URLs defined in the [Server Object](#server-object) as a Base URL. Note that these themselves MAY be relative to the referring document.
+Unless specified otherwise, relative references are resolved using the URLs defined in the [Server Object](#server-object) as a Base URL. Note that these themselves MAY be relative to the referring document (**NOT** the [OpenAPI Object's](#openapi-object) `$self` field).
```

Comment:
add "..., which may itself be relative)" to the end of this sentence?

Reply:
@karenetheridge I'm not sure what you mean, we already say that the Server Object URLs may be relative, and the point of the parenthetical aside is that `$self` URI is irrelevant here, so adding more information about `$self` would just be a distraction and make it feel relevant.

Reply:
Okay. I originally actually overlooked the "$self is not used here" part entirely as it wasn't what I was expecting.

That means that the referring URL of the document (if there is one) holds relevance to more than just $self - other things are resolved against it as well.

I'll take up the rest of this in your new examples.